dev/com.ibm.ws.crypto.passwordutil/src/com/ibm/websphere/crypto/PasswordUtil.java

```diff
@@ -744,8 +744,9 @@ public static String encode_password(String decoded_string, String crypto_algori
                     }
                 }
             }
-
-            buffer.append(crypto_algorithm);
+            // if "aes-128" or "aes-256" is used we want to make the tag "aes" instead so it will be usable on Liberty servers older than 25.0.0.4
```

Comment:
`aes-256` in the comment should just be `aes` because:
- `encipher_internal` only supports `AES` and `AES_128`
- the description in `UtilityOptions.nlsprops` only mentions `aes` and `aes-128`

perhaps the code can also be updated to be:
```
String normalizedCryptoAlgorithm = crypto_algorithm.equalsIgnoreCase("aes-128") ? "aes" : crypto_algorithm;
```

Reply:
I am doing this intentionally for future support